From ca579cc5ce9e3fa65f66c1cb8aa031f7b3963eec Mon Sep 17 00:00:00 2001
From: ArchanaShinde1 <archana.shinde2504@gmail.com>
Date: Tue, 23 Jan 2024 10:36:46 +0000
Subject: [PATCH] abseil-cpp

---
 distdir_deps.bzl | 9 ++++-----
 1 file changed, 4 insertions(+), 5 deletions(-)

diff --git a/distdir_deps.bzl b/distdir_deps.bzl
index 2ca988e260..c378ef6f9a 100644
--- a/distdir_deps.bzl
+++ b/distdir_deps.bzl
@@ -162,17 +162,16 @@ DIST_DEPS = {
         ],
     },
     "com_google_absl": {
-        "archive": "20211102.0.tar.gz",
-        "sha256": "dcf71b9cba8dc0ca9940c4b316a0c796be8fab42b070bb6b7cab62b48f0e66c4",
+        "archive": "20230802.0",
+        "sha256": "59d2976af9d6ecf001a81a35749a6e551a335b949d34918cfade07737b9d93c5",
         "urls": [
-            "https://mirror.bazel.build/github.com/abseil/abseil-cpp/archive/refs/tags/20211102.0.tar.gz",
-            "https://github.com/abseil/abseil-cpp/archive/refs/tags/20211102.0.tar.gz",
+            "https://github.com/abseil/abseil-cpp/archive/refs/tags/20230802.0.tar.gz",
         ],
         "used_in": [
             "additional_distfiles",
             "test_WORKSPACE_files",
         ],
-        "strip_prefix": "abseil-cpp-20211102.0",
+        "strip_prefix": "abseil-cpp-20230802.0",
     },
     "zstd-jni": {
         "archive": "v1.5.0-4.zip",
-- 
2.40.1

